tests/autopilot/autopilot_test.go

```diff
@@ -407,10 +407,10 @@ var _ = Describe(fmt.Sprintf("{%sPoolExpand}", testSuiteName), func() {
 		}
 
 		poolExpandLabels := map[string]map[string]string{
-			"addDiskLabel":          map[string]string{"autopilot": "adddisk"},
-			"addDiskFixedSizeLabel": map[string]string{"autopilot": "adddiskfixedsize"},
-			"resizeDiskLabel":       map[string]string{"autopilot": "resizedisk"},
-			"resizeFixedSizeLabel":  map[string]string{"autopilot": "resizefixedsize"},
+			"addDiskLabel":          {"autopilot": "adddisk"},
```

Comment:
why are we touching autopilot-related test files for longevity modifications? If it's a linting issue, then can we raise a separate PR for it?

Reply:
Revoked these auto-pilot changes